UPGRADING.md

```diff
@@ -1,8 +1,14 @@
 # Upgrade guide
 
+# Upgrading from 6.x to 7.x
+
+Version 7 of the Embrace Android SDK has breaking changes:
+
+- The `startMoment/endMoment` API has been removed. Use `startSpan/recordSpan` instead.
```

Comment:
We should probably write a similar upgrade guide that iOS has for moving from Moments to Traces

Reply:
Added docs here: https://github.com/embrace-io/embrace-docs/pull/43